apps/aechannel/src/aesc_session_noise.erl

```diff
@@ -134,10 +162,10 @@ tcp_opts(_Op, Opts) ->
 
 
 noise_defaults() ->
-    [{protocol, <<"Noise_XK_25519_ChaChaPoly_BLAKE2b">>}].
+    [{noise, <<"Noise_XK_25519_ChaChaPoly_BLAKE2b">>}].
 
 tcp_defaults() ->
-    [{active, once},
+    [{active, true},
```

Comment:
Probably not an issue here, but I've had trouble when I was delegating the socket and messages arrived at the wrong process (too early). Starting with active false and activate once setup is done is an alternative.

Reply:
Yes, `{active, once}` clearly didn't work. I decided that I don't care what `enoise` wants during the setup, and can set `{active, once}` after successful noise negotiation. I'll probably change to `{active, false}` in a future PR then.